.gitignore

```diff
@@ -20,4 +20,6 @@ tmp
 *.o
 *.a
 mkmf.log
-gemfiles/*.lock
\ No newline at end of file
+gemfiles/*.lock
+*.ruby-*
```

Comment:
Why did you add this to `gitignore`?
I think they are good to have because all developers will be on the same version and thus discussion wont be affected by different behaviours through ruby versions.
And we can always trust `travis` to test all versions for us. =)